lib/util/logger.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const chalk = require('chalk')
+
+exports.error = msg => console.error(chalk.red(msg))
+
+exports.success = msg => console.log(chalk.green(msg))
+
+exports.info = msg => console.log(msg)
```

Comment:
I think we can use `console.log` directly instead. Let's keep the `logger` utility for colored output.

Reply:
okey.